docker/pulp/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:7
+FROM centos:8
```

Comment:
Changing major version of container is probably out of scope of this change?

Reply:
Moving this to a separate PR.